### ISSUE TITLE
Error when opening project

### ISSUE THREAD

**Issue by joseph-illingworth** [bardsoftware/ganttproject#1287]
Something went wrong 

An attempt to set length=-1591.0 day to task=2019

I'm 40 hours into this project and something flipped out and now only half the tasks are loading. Log:

C:\Users\jillingworth\ganttproject.log

GanttProject 2.8 Pilsen (build 2016)
Settings file:
    location: C:\Users\jillingworth.ganttproject
    size:4905
    is readable: true
java.class.path: eclipsito.jar;.
java.home: C:\Program Files (x86)\Java\jre1.8.0_91
java.ext.dirs: C:\Program Files (x86)\Java\jre1.8.0_91\lib\ext;C:\Windows\Sun\Java\lib\ext
java.io.tmpdir: C:\Users\JILLIN~1\AppData\Local\Temp\
java.runtime.version: 1.8.0_91-b14
java.vendor: Oracle Corporation
java.vm.name: Java HotSpot(TM) Client VM
java.vm.vendor: Oracle Corporation
java.vm.version: 25.91-b14
os.arch: x86
os.name: Windows 7
os.version: 6.1
sun.java.command: org.bardsoftware.eclipsito.Boot ganttproject-eclipsito-config.xml -log true
user.country: US
user.dir: C:\Program Files (x86)\GanttProject-2.8
user.home: C:\Users\jillingworth
user.language: en
user.timezone: America/Denver

Creating main frame...
1. loading look'n'feels
no editor for column=0 class=interface javax.swing.Icon
no editor for column=1 class=interface javax.swing.Icon
no editor for column=2 class=interface javax.swing.Icon
2. loading options
3. creating menus...
4. creating views...
5. calculating size and packing...
6. changing language ...
7. changing look'n'feel ...
8. finalizing...
Main frame created
Splash closed
no editor for column=0 class=interface javax.swing.Icon
no editor for column=1 class=interface javax.swing.Icon
no editor for column=2 class=interface javax.swing.Icon

**Comment by dbarashev** [comment]
Please share the full log (what you posted seems to be the very beginning of the log) and your project file.

**Comment by joseph-illingworth** [comment]
dbarashev,

My apologies. I thought that was the log. I have saved a full copy as a text file and zipped it with a copy of the project file using 7Zip. the project file size indicates that all the data should still be there, but it doesn't display upon loading. 

[GanntProject.zip](https://github.com/bardsoftware/ganttproject/files/386539/GanntProject.zip)

**Comment by joseph-illingworth** [comment]
Hey dbarashev,

I'm not a programmer by any stretch, but I was able to change the extension on the project to .txt and open it in a text editor. I searched for duration="- to find the negative duration that was giving me the error and removed the negative from the task duration. I then switched the project back to .gan and it appears to be working currently. 

I'm not sure why the program set a negative duration for those tasks, it certainly wasn't something I had done. If there's a function that's automatically checking and adjusting task duration length upon startup, I would look there.
